src/app/components/team/Newsletter/NewsletterComponent.js

```diff
@@ -468,6 +468,7 @@ const NewsletterComponent = ({
               handleFileChange={handleFileChange}
               setFile={setFile}
               setFileName={setFileName}
+              hideOptions={['link_preview']}
               availableHeaderTypes={team.available_newsletter_header_types || []}
```

Comment:
Does it work if you change the existing value passed to `availableHeaderTypes` instead of adding a new `prop`?

```
availableHeaderTypes={team.available_newsletter_header_types ? team.available_newsletter_header_types.filter(type => type !== 'link_preview') : []}
```